apps/editing-toolkit/editing-toolkit-plugin/block-patterns/class-block-patterns.php

```diff
@@ -21,11 +21,35 @@ class Block_Patterns {
 	 */
 	private static $instance = null;
 
+	/**
+	 * Cache key for patterns array.
+	 *
+	 * @var string
+	 */
+	private $patterns_cache_key;
+
+	/**
+	 * Patterns source site ID.
+	 *
+	 * @var int
+	 */
+	private $source_site_id = 174455321;
```

Comment:
We don't need to specify the source site for block patterns (the API defaults to dotcompatterns so that consumers don't have to worry about it), so I think this can be removed.